### ISSUE TITLE
usb serial on android receives data from Arduino but won't connect to unity

### ISSUE THREAD

**Issue by ghhovagimyan** [dwilches/Ardity#51]
Android has an OTG cable attached to my Arduino data glove.  The usb serial app on android shows the data stream is being received. Using Ardity this works in Unity on OS X when I compile to Android it doesn't work.  I will mention that the physical data glove is connected to a virtual hand.  The gyro info and the finger movement data work in the Unity workspace.  I thought it might be a naming convention for the USB port.   After much searching I found that my original address for the Data glove /dev/cu.Repleo-CH341-00101114 was different that the android port. So I changed the port to /dev/bus/usb/001/002 
Still no connection.  Various usb apps on android named the chip CH34x and also MPU6050. But Unity is reading it.  
The nature of the chip is to communicate with Baud 115200. Android receives the info this way and it's clear.   So I'm stumped?

**Comment by dwilches** [comment]
Hello,

I also thought the problem was the port name, but if you're sure the port name is ok, some things to try:
* Can you tell your device to communicate at a lower baudrate than 115200?
* Is the "Api Compatibility Level" set to ".Net 4.0" instead of ".Net Subset"?

Regards

**Comment by ghhovagimyan** [comment]
I've tried to change net 4.0 to standard 2.0. I get this build error in Unity "Assets/Ardity/Scripts/Threads/SerialThread.cs(11,17): error CS0234: The type or namespace name 'Ports' does not exist in the namespace 'System.IO' (are you missing an assembly reference?)"

**Comment by ghhovagimyan** [comment]
I've tried to change net 4.0 to standard 2.0. I get this build error in Unity "Assets/Ardity/Scripts/Threads/SerialThread.cs(11,17): error CS0234: The type or namespace name 'Ports' does not exist in the namespace 'System.IO' (are you missing an assembly reference?)"
Also I've lowered the baud rate to 9600.  Still nothing.

**Comment by dwilches** [comment]
That error is a [known issue](https://github.com/dwilches/Ardity#the-type-or-namespace-name-ports-does-not-exist-in-the-namespace-systemio) when a reduced .Net runtime is selected. Try with other `API Compatibility Level` to see which one works for you.

**Comment by ghhovagimyan** [comment]
I have set the API Level to .NET 4.0.  The data glove streams serial info that is read by various Android apps such as; usb monitor or UArtBridge.  I set the  port to /dev/bus/usb/001/002 in Unity.   no data stream comes into Unity.   I'm wondering whether this is because I'm using an OTG cable to connect the dataglove to the Android phone.  The only thing I can think of is the the path to the port on the glove must be added.
